src/transcoding/transcode_ext_handler.cc

```diff
@@ -77,11 +77,12 @@ std::unique_ptr<IOHandler> TranscodeExternalHandler::serveContent(const std::sha
 #endif
 
     std::vector<std::shared_ptr<ProcListItem>> procList;
+    fs::path loc2 = location;
 
     bool isURL = obj->isExternalItem();
     if (isURL && !profile->acceptURL()) {
 #ifdef HAVE_CURL
-        openCurlFifo(location, procList);
+        openCurlFifo(loc2, procList);
```

Comment:
In that case change openCurlFifo to fs::path openCurlFifo(const fs::path& location, ..) so that the call is `auto fifoLocation = openCurlFifo(location, procList);` make the code more obvious.

Reply:
Reworked. I think this is what you meant.